creativecloud/blocks/catalog/catalog.js

```diff
@@ -85,6 +103,7 @@ export default async function init(el) {
       const sidenav = await initSidenav(sidenavEl);
       el.prepend(sidenav);
       await sidenav.updateComplete;
+      addCheckboxGroupAttributes(sidenav);
```

Comment:
@bozojovicic 
I think this should be addressed in milo at https://github.com/adobecom/milo/blob/stage/libs/features/mas/src/sidenav/merch-sidenav-checkbox-group.js

Reply:
Right. Agreed. And again, last week I tried to do that and I gave up and now I don't remember why. I created Milo PR for that https://github.com/adobecom/milo/pull/3518 and it works quite nice. So closing this one.